src/pages/academy/gameSimulator/subcomponents/assetViewer/AssetViewerUpload.tsx

```diff
@@ -51,32 +46,28 @@ const AssetFileUploader = () => {
       {s3AssetFolders.map(folder => (
         <MenuItem onClick={handleChangeUploadFolder} id={folder} key={folder} text={folder} />
       ))}
-      <MenuItem
-        onClick={showSpecifyFolder}
-        id={specifyFolderText}
-        key={specifyFolderText}
-        text={specifyFolderText}
-      ></MenuItem>
+      <MenuItem onClick={showSpecifyFolder} text={'Custom Folder'}></MenuItem>
     </Menu>
   );
   return (
     <div className="LeftAlign">
-      <h4>Choose Folder</h4>
-      <Popover2 placement={Position.BOTTOM} content={uploadButtonPopoverContent}>
+      <h4>Choose a Folder to upload an asset to:</h4>
+      <Popover placement={Position.BOTTOM} content={uploadButtonPopoverContent}>
         <Button text={uploadFolder} />
-      </Popover2>
+      </Popover>
+      <br />
+      <br />
       {showfolderOverwrite && (
         <InputGroup
-          placeholder={folderOverwritePlaceholder}
+          placeholder={"Specify your own custom folder: e.g. 'locations/hallway'"}
```

Comment:
Nit: Unnecessary curly braces around string.

Reply:
Addressed in [d6b805d](https://github.com/source-academy/frontend/pull/2836/commits/d6b805d1a82065dea06d44dae8eff4438b1abcf7).